### PR TITLE
Do not cache csv and pdf in service worker

### PR DESCRIPTION
**Pull request by krzysztofwolski**
I want to merge this change because...
storefront is hijacking csv files export.

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] Changes are mentioned in the changelog.

### Test Environment Config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by krzysztofwolski** [comment]
I'm getting a different kind of 404 now, not intercepted by storefront worker.
https://fix-invoice-download-issue.storefront.saleor.rocks/media/export_files/product_data_04_08_2020.csv

File: src/app/routes/AppRoutes.tsx [hunk 1]
```diff
@@ -35,6 +37,7 @@ export const Routes: React.FC = () => (
     <Route path={paths.passwordResetUrl} component={PasswordReset} />
     <Route path={paths.checkoutUrl} component={CheckoutPage} />
     <Route path={paths.orderFinalizedUrl} component={ThankYouPage} />
+    <Route path="/media/*" onEnter={reload} />
```
Comment (patrys) [inline src/app/routes/AppRoutes.tsx:40]: This feels like a hack, won't it cause any refresh loops?

File: src/sw.js [hunk 2]
```diff
@@ -10,13 +10,17 @@ workbox.core.setCacheNameDetails({
...
 );
 
 workbox.routing.registerNavigationRoute(
   workbox.precaching.getCacheKeyForURL("/index.html"),
   {
-    blacklist: [new RegExp("/graphql"), new RegExp("/dashboard")],
+    blacklist: [
+      new RegExp("/graphql"),
+      new RegExp("/dashboard"),
+      new RegExp("/media"),
```
Comment (patrys) [inline src/sw.js:23]: What about images that live under media? That includes all product images.

**Comment by patrys** [comment]
Task linked: [CU-74pghn Storefront worker causing issues with csv download](https://app.clickup.com/t/74pghn)

**Pull request merged**